doozer/doozerlib/cli/__main__.py

```diff
@@ -280,6 +280,8 @@ def beta_reposync(runtime, output, cachedir, arch, repo_type, names, dry_run):
                 continue
 
             color_print('Syncing repo {}'.format(repo.name), 'blue')
+            if arch == "x86_64":
+                arch = arch + ' --arch noarch '
```

Comment:
The naming of the variable will be misleading: It is named "arch", but it is actually part of command line options.